src/app/view/device/add-devices/add-devices.component.html

```diff
@@ -384,6 +376,34 @@ <h3 class="pb-0 mb-0 fw-bold text-capitalize">
                 </div>
 
                 <div class="example-container1 row" *ngIf="addmoredetals[i]">
+                  <div class="col-md-3">
+                    <mat-form-field appearance="outline">
+                      <mat-label class="text-capitalize"
+                        >Postal Code (Zip Code)</mat-label
+                      >
+                      <input
+                        matInput
+                        type="string"
+                        formControlName="postalCode"
+                        test-id="postal-code"
+                      />
+                      <mat-error>Please enter the valid postal code</mat-error>
+                    </mat-form-field>
+                  </div>
+                  <div class="col-md-3" *ngIf="showinput[i]">
```

Comment:
Please avoid changing the order of other fields and their position unless specified by the ticket